.github/workflows/run_unittests.yml

```diff
@@ -0,0 +1,49 @@
+
+name: Vectorflow_Unittests
+
+on:
+  pull_request:
+    branches: [ "main" ]
+    types: ["opened", "reopened", "edited"]
+
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: ["3.9", "3.10"]
+
+    env:
+      LOCAL_VECTOR_DB: ${{ secrets.LOCAL_VECTOR_DB }}
```

Comment:
set this equal to `qdrant` if the value does not matter. let's not use secrets where they are not needed.

Reply:
I've changed the value. Everything still works @dgarnitz

Reply:
Awesome. thank you!